tests/cpu/test_ckf_sparse_tracks_telescope.cpp

```diff
@@ -66,7 +68,23 @@ TEST_P(CkfSparseTrackTests, Run) {
     tel_cfg.mat_thickness(thickness);
     tel_cfg.pilot_track(traj);
 
-    auto [host_det, name_map] = create_telescope_detector(host_mr, tel_cfg);
+    // Create telescope detector
+    auto [det, name_map] = create_telescope_detector(host_mr, tel_cfg);
```

Comment:
Um no. I would not like to maintain detector files separately. And I think having different test classes is not bad because we can change the configuration parameters independently